### DISCUSSION TITLE
Real-time chat for the executablebooks community?

### DISCUSSION THREAD

**Discussion by iso2013** [executablebooks/meta#1170] [General] [answered]
Hello, thanks for all your work on these projects!

Is there any sort of chat for interacting with project contributors?

I'm a fairly-capable programmer and I've got some time I can dedicate to these projects over the next few months. I'd like to start contribute fixes for a few bugs I've noticed, and fix some issues on various repositories as well. However, due to the complexity of the jupyter-book project, it's hard to know where to start looking for an issue's cause. Further, I'm not sure what sort of changes would be preferred (like updating the Python requirement for Jupyter-Book to 3.12 to fix an import issue, or manually checking compatibility and leaving the requirement as Python 3.9). Being able to ask & answer questions of this kind would be fantastic.

**Comment by lrq3000** [comment]
@rowanc1 May be interested, and I think the team is preparing for SciPy 2024 so there is a lot going on I guess, exciting times for the project :D

I don't know if such a server exists, but if not, I would suggest creating a Matrix room, they work superbly well for team chats (they include threads).

**Comment by choldgraf** [comment, marked as answer]
Thanks so much for being interested in the project! Right now there are two broad focus areas and technical stacks associated with executable books.

**A JavaScript and jupyter-focused stack** that is built around the new MyST document engine here: https://mystmd.org . I suspect that jupyter book will eventually use this as it's back end. This part of the project has a [discord channel for real time chat here](https://discord.mystmd.org/), and [a place for asynchronous questions and discussion here](https://github.com/executablebooks/mystmd/discussions). You and anyone else is invited to join! It is about a year old in heavy development, and where a lot of the energy in the project is right now.

**A Sphinx-based stack** that has been under development for 3 years, and has powered jupyter book historically. I suspect the sphinx stack will continue to develop but with more focus on developer workflows (which is what Sphinx is built for). This does not have a real time chat space that I know of but you can use this discussions board to ask questions if you wish!
